src/Data/CurrentUser/errorToast.tsx

```diff
@@ -22,3 +22,9 @@ export function errorToast(title: string, error: unknown): void {
     },
   )
 }
+
+export function simpleErrorToast(title: string): void {
+  console.error('simpleErrorToast', title)
```

Comment:
* [ ] FYI having `console.error` in the `errorToast` helpers may be unexpected for callers. The helpers look like a simple convenience for `toast.error` on the first sight. But actually they may trigger error reporting. Maybe we should rename them to `reportError` or so. Or remove the console logging.